tests/test_pytest_cov.py

```diff
@@ -343,6 +343,20 @@ def test_cov_min_50(testdir):
     ])
 
 
+def test_cov_min_float_value(testdir):
+    script = testdir.makepyfile(SCRIPT)
```

Comment:
Would be good to get a test that actually passes/fails based on a fraction of a percentage coverage